src/main/java/org/apache/cassandra/sidecar/cluster/InstancesConfig.java

```diff
@@ -22,15 +22,20 @@
 import java.util.NoSuchElementException;
 
 import org.apache.cassandra.sidecar.cluster.instance.InstanceMetadata;
+import org.jetbrains.annotations.NotNull;
 
 /**
  * Maintains metadata of instances maintained by Sidecar.
  */
 public interface InstancesConfig
 {
     /**
+     * Returns metadata associated with the Cassandra instances managed by this Sidecar. The implementer
```

Comment:
Clarification for the implementer that the instances list must not be null. An empty collection is acceptable